### PR TITLE
Allow horizontal scrolling with left/right arrow keys.

### PR DESCRIPTION
**Pull request by glennw**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/9359)

### CODE REVIEW

**Comment by glennw** [comment]
This is needed to debug some of the scrolling issues in browser.html on linux - such as https://github.com/servo/servo/issues/9306.

File: ports/glutin/window.rs [hunk 1]
```diff
@@ -702,6 +702,12 @@ impl WindowMethods for Window {
             (NONE, Key::Down) => {
                 self.scroll_window(0.0, -3.0 * LINE_HEIGHT);
             }
+            (NONE, Key::Left) => {
+                self.scroll_window(LINE_HEIGHT, 0.0);
+            }
+            (NONE, Key::Right) => {
+                self.scroll_window(-LINE_HEIGHT, 0.0);
```
Comment (KiChjang) [inline ports/glutin/window.rs]: Should these be `3.0 * LINE_HEIGHT` (similar to the Up and Down keys above)?

Comment (glennw) [inline ports/glutin/window.rs]: I think it's preferable for the horizontal scroll to be more fine grained than the vertical scroll, by default.

**Pull request merged**